xwiki-platform-core/xwiki-platform-tag/xwiki-platform-tag-ui/src/main/resources/XWiki/TagCloud.xml

```diff
@@ -338,8 +347,25 @@ ol.tagCloud .ultraPopular {
 ##
 $xwiki.ssx.use('XWiki.TagCloud')##
 ##
-## Get tag count map. Key: tag, value: number of occurrences.
-#set ($tagCount = $xwiki.tag.getTagCount("$!tagCloudSpace"))
+##if (!("$!tagCloudSpace" =="") &amp;&amp; !("$!tagCloudSpaces" ==""))
+#if( $tagCloudSpace &amp;&amp; $tagCloudSpaces )
+{{html}}
```

Comment:
Look like what I was expecting yes :)

What Vincent meant is that it would be nicer to make explicit in the project pom.xml that you require error macro (org.xwiki.rendering:xwiki-rendering-macro-message) but it's more being a good citizen than a technical requirement since this macro will probably be part of the minimal WAR for long.